src/panels/config/automation/ha-automation-editor.ts

```diff
@@ -502,9 +502,9 @@ export class HaAutomationEditor extends KeyboardShortcutMixin(LitElement) {
         }
       },
       (errors) => {
-        this._errors = errors.body.message;
+        this._errors = errors.body.message || errors.body;
```

Comment:
?
```suggestion
        this._errors = errors.body.message || errors.error || errors.body;
```

Reply:
I had the body first, since it provides a bit more context (at least in this example), but I don't mind either order.